packages/utils/project.json

```diff
@@ -29,6 +29,10 @@
         ]
       }
     },
+    "perf": {
+      "command": "npx node ./packages/utils/perf/index.mjs",
+      "dependsOn": ["lint"]
```

Comment:
Why would it depend on `lint`? :confused:

Reply:
Good catch! Was a mistake.

Reply:
done